theatre/studio/src/uiComponents/form/BasicNumberInput.tsx

```diff
@@ -133,17 +131,22 @@ const BasicNumberInput: React.FC<{
     stateRef.current.mode === 'editingViaKeyboard',
   )
 
-  const bodyCursorBeforeDrag = useRef<string | null>(null)
-
   const callbacks = useMemo(() => {
+    const isValid = propsA.isValid ?? alwaysValid
+    let inputWidth: number
+
     const inputChange = (e: React.ChangeEvent) => {
       const target = e.target as HTMLInputElement
       const {value} = target
       const curState = stateRef.current as IState_EditingViaKeyboard
 
-      stateRef.current = {...curState, currentEditedValueInString: value}
+      stateRef.current = {
+        ...curState,
+        currentEditedValueInString: value,
+      }
 
```

Comment:
@AriaMinaei should the value when `editingViaKeyboard` take into consideration the `range` prop?